meteor/server/api/templates/templates.ts

```diff
@@ -70,6 +70,9 @@ export type SegmentLineItemOptional = Fix<SegmentLineItem>
 export type SegmentLineItemOptional = Optional<SegmentLineItem>
 export type SegmentLineAdLibItemOptional = Optional<SegmentLineAdLibItem>
 export type RunningOrderBaselineItemOptional = Optional<RunningOrderBaselineItem>
+export type TemplateRuntimeArguments = {
+	[key: string]: string
```

Comment:
(optional/question) could this be a number also? or _any_?

Reply:
I guess so. I don't know enough about the new blueprint system to know if that makes sense. With plain JavaScript & the value of the Runtime argument being essentially user input (it's just two text fields in the settings), I think assuming this is a string is a safer choice.

Reply:
agreed :)